jobs/furnishings/src/furnishings/stage_processors/post_processor.py

```diff
@@ -103,7 +104,7 @@ def __init__(self, app, furnishings_dict):
             username=app.config.get('BCLAWS_SFTP_USERNAME'),
             host=app.config.get('BCLAWS_SFTP_HOST'),
             port=app.config.get('BCLAWS_SFTP_PORT'),
-            private_key=app.config.get('BCLAWS_SFTP_PRIVATE_KEY'),
+            private_key=base64.b64decode(app.config.get('BCLAWS_SFTP_PRIVATE_KEY')).decode('utf-8'),
```

Comment:
To encode the private key for the `.env` file, I ran the following command on the generated private key file.
```
openssl base64 -A -in keys/id_ed25519 -out keys/id_ed25519_encoded
```

This resulted in a single line encoded string, which I copied into the `.env`. I also updated the `Makefile` in my sftp-server repo to [include these commands](https://github.com/leodube-aot/sftp-server/blob/main/Makefile#L10-L12).